earn/src/components/boost/ImportModal.tsx

```diff
@@ -175,77 +259,147 @@ export default function ImportModal(props: ImportModalProps) {
     confirmations: 1,
     hash: mintTxn?.hash,
     chainId: activeChain.id,
-    onSuccess(data) {
-      // TODO: prob want to reload the page in here. keep the console.debug if possible (for consistency since we have this style of thing elsewhere)
-      console.debug('Mint transaction successful!', data);
+    onSuccess() {
+      navigate(0);
     },
   });
 
-  // TODO: you can remove these console.logs obviously
-  let state: ImportModalState;
+  let state: ImportModalState | undefined;
```

Comment:
Why/when would this be undefined?

Reply:
It has no default value and all of the places that set it are within if statements (no else)

Reply:
Default should probably be the loading state then

Reply:
There is no "loading state", we just treat no state as loading state which is technically the same thing.